client/lib/upgrades/actions/purchases.js

```diff
@@ -150,11 +150,18 @@ function fetchUserPurchases() {
 	} );
 }
 
+function invalidatePurchases() {
+	Dispatcher.handleViewAction( {
+		type: ActionTypes.PURCHASES_INVALIDATE
+	} );
+}
+
```

Comment:
I named this differently because the mechanism is slightly different (the old purchases stay in memory). There isn't a good reason for that though, so I'll update the name and the behavior of the reducer to actually remove the stale purchases from memory. :+1: